pkg/handler/handler_test.go

```diff
@@ -201,11 +192,19 @@ func p(i int32) *int32 {
 	return &i
 }
 func TestReadmeUtgaver(t *testing.T) {
-	r := strings.NewReader(in)
-	readmes, _ := parseReadmes(r)
-	sortReadmes(&readmes)
+	if offline == nil || *offline {
+		t.Skip()
```

Comment:
Why?

Reply:
Because this this calls the google api. There is no reason not to, but there was the option before, so I kept it.

Reply:
Hmmm. Would still be nice with some real tests not calling the API in order to make it easier to understand what is happening.

Would be nice with some docs about the format then, since no tests show how that works.

Reply:
I agree that we should have some tests for that, but currently that's not directly possible (in a good way). But I could add some docs.